ui/packages/catalog-ui-search/src/main/webapp/styles/components/global-bracket.less

```diff
@@ -2,6 +2,11 @@
 .global-bracket.is-right {
   position: relative;
 }
+
+.global-bracket.is-right {
```

Comment:
Can this be combined with the one above it?

Reply:
No, I don't want that calculated width to apply to left brackets like filters